lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+  attr_accessor :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron_name)
+    # Add addition patrons to the venue list.
+    @patrons << patron_name
+  end
+
+  def yell_at_patrons
+    # Iterate through patrons and change string value to upcase.
+    @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    # If the number of patrons is greater than capacity return true.
+    @patrons.count > @capacity ? true : false
```

Comment:
Nice use of a ternary here. You could also just write it like:
```
@patrons.count > @capacity
```
the `>` operator will return a boolean